### ISSUE TITLE
[Bug]: disabled Link does not have enough constrast

### ISSUE THREAD

**Issue by layershifter** [microsoft/fluentui#23330]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
@fluentui/react-components@rc-13
```

### Are you reporting Accessibility issue?

no

### Reproduction

N/A

### Bug Description

- Go to https://react.fluentui.dev/?path=/docs/components-link--default
- Select Dark theme
- Scroll to "Disabled"

## Actual Behavior
![image](https://user-images.githubusercontent.com/14183168/171145014-8b3b648f-eb05-4229-853f-c90e3c9b481e.png)

Contrast is not sufficient:
![image](https://user-images.githubusercontent.com/14183168/171145222-87d0ba28-a391-42bd-b46a-bfb3f2a2b891.png)

## Expected Behavior
Decent contrast ratio :)

### Logs

```shell
N/A
```

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by tudorpopams** [comment]
@microsoft/cxe-red is this still an issue? If so, we should fix.

**Comment by micahgodbolt** [comment]
disabled text does not need to meet the same contrast ratio as enabled text. I'll check with Sarah on if there is a minimum we SHOULD be hitting

**Comment by smhigley** [comment]
There's no actual hard minimum for disabled controls' text, mostly there isn't a lot of agreement on how to spec a minimum without interfering with the ability to tell a disabled control from an enabled one.

Low contrast text will still be hard to read in practice, so I'm generally in favor of increasing contrast. Since it'll probably still not be up to the 4.5:1 contrast minimum needed for text, we should still give out author guidance that disabled controls should never need to be perceived for a user to understand and use a page.

**Comment by sopranopillow** [comment]
I will be closing this issue since it's not really a bug. I double checked with Sarah offline.
